tests/run/cpp_operator_exc_handling.srctree

```diff
@@ -0,0 +1,485 @@
+# tag: cpp
+
+"""
+PYTHON setup.py build_ext --inplace
+PYTHON -c "from check_operator_exception_handling import test; test()"
+"""
+
+######## setup.py ########
+
+from distutils.core import setup
+from Cython.Build import cythonize
+setup(ext_modules=cythonize('*.pyx', language='c++'))
+
+######## wint.cpp ########
```

Comment:
No need for this to be a srctree test, could you instead make it a standard test (if you have foo.pyx, any foo_helper.cpp will be available as well).